net-analyzer/shodan/shodan-1.28.0.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{8..10} )
+inherit distutils-r1
+
```

Comment:
I haven't checked yet, so apologies, but does this work standalone (does it provide some CLI tool?) or is it "just" a Python library? (If the latter, we should put it in dev-python)

Reply:
Both packages provide a command line tools in addition to a python library.